lib/database/models/qualityControlFlag.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const Sequelize = require('sequelize');
+
+module.exports = (sequelize) => {
+    const QualityControlFlag = sequelize.define(
+        'QualityControlFlag',
+        {
+            id: {
+                type: Sequelize.INTEGER,
+                allowNull: false,
+                primaryKey: true,
+                autoIncrement: true,
+            },
+            timeStart: {
+                type: Sequelize.DATE,
+                allowNull: true,
+            },
+            timeEnd: {
+                type: Sequelize.DATE,
+                allowNull: true,
+            },
```

Comment:
If it's a period, I prefer `from` and `to`, but it's up to you

Reply:
well, I would keep to have it consistent with runs columns names, as it is related

Reply:
A run is actually started and stopped, it makes sense in its context.
A flag is not started neither stopped, it simply spans across a given period. So, please use `from` and `to` instead of `timeStart` and `timeEnd`.

Reply:
applied